packages/blockchain-wallet-v4-frontend/src/scenes/Login/Exchange/TwoFA/index.tsx

```diff
@@ -91,7 +91,7 @@ const TwoFAExchange = (props: Props) => {
             nature='primary'
             fullwidth
             height='48px'
-            disabled={submitting || invalid || busy || !formValues?.exchangeTwoFA}
+            disabled={submitting || invalid || busy || (formValues?.exchangeTwoFA?.length || 0) < 5}
```

Comment:
shouldn't a 2FA length be 6?

Reply:
This page is only for legacy exchange login which only uses auth app, so let's bump to 6. Wallet/TwoFA should be 5.